docs/aws/audit/elbmonitoring/rules/classic_elb_listeners_acm_certificate.mdx

```diff
@@ -23,6 +23,202 @@ CBP,RBI_UCB,RBI_MD_ITF
+
+5. Check the SSL Certificate column. If the SSL Certificate is set to "None" or if there is no ACM (AWS Certificate Manager) certificate listed, then the Classic ELB does not have at least one ACM certificate.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. First, you need to install and configure AWS CLI on your local machine. You can do this by following the instructions provided by AWS. Make sure you have the necessary permissions to access the Elastic Load Balancer (ELB) resources.
+
+2. Once the AWS CLI is set up, you can list all the Classic Load Balancers in your account using the following command:
+
+   ```
+   aws elb describe-load-balancers
+   ```
+
+   This command will return a JSON output with details about all the Classic Load Balancers in your account.
+
+3. To check the listeners for a specific Classic Load Balancer, you can use the following command:
+
+   ```
+   aws elb describe-load-balancer-policies --load-balancer-name <your-load-balancer-name>
+   ```
+
+   Replace `<your-load-balancer-name>` with the name of your Classic Load Balancer. This command will return a JSON output with details about the policies attached to the specified Classic Load Balancer.
+
+4. Now, you need to check if there is at least one ACM (AWS Certificate Manager) certificate attached to the listeners of the Classic Load Balancer. You can do this by looking for the "SSLCertificateId" field in the JSON output from the previous step. If the value of this field starts with "arn:aws:acm:", it means that an ACM certificate is attached to the listener. If no such field is found, it means that there is no ACM certificate attached to the listeners of the Classic Load Balancer.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install and configure AWS SDK for Python (Boto3) on your local system. Boto3 allows you to directly create, update, and delete AWS services from your Python scripts.
+
+2. Import the necessary modules and create a session using your AWS credentials. 
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
+    region_name='us-west-2'
+)
+```
+
+3. Create an ELB client using the session and retrieve all the load balancers.
+
+```python
+elb = session.client('elbv2')
+
+response = elb.describe_load_balancers()
+```
+
+4. Iterate over the load balancers and check the listeners for each load balancer. If a listener does not have an ACM certificate, print a message.
+
+```python
+for lb in response['LoadBalancers']:
+    listeners = elb.describe_listeners(LoadBalancerArn=lb['LoadBalancerArn'])
+    for listener in listeners['Listeners']:
+        if 'Certificates' not in listener or not any(cert['IsDefault'] for cert in listener['Certificates']):
+            print(f"Load balancer {lb['LoadBalancerName']} has a listener without an ACM certificate.")
+```
+
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>